features/admin.extensions.v1/components/groups/wizard/group-assign-users.tsx

```diff
@@ -165,7 +165,7 @@ export const AddGroupUsers: FunctionComponent<AddGroupUserProps> = (props: AddGr
     };
 
     useEffect(() => {
-        setListItemLimit(UIConstants.DEFAULT_RESOURCE_LIST_ITEM_LIMIT);
+        setListItemLimit(null);
```

Comment:
Shall we initialize `listItemLimit` variable with `null` and get rid of this setState call?